src/components/Editor.tsx

```diff
@@ -355,6 +355,31 @@ export const Editor = ({ document, puckConfigs }: EditorProps) => {
     TARGET_ORIGINS
   );
 
+  const { sendToParent: openQuickFind } = useSendMessageToParent(
+    "openQuickFind",
+    TARGET_ORIGINS
+  );
+
+  const keyboardHandler = (event: KeyboardEvent) => {
+    if ((event.ctrlKey || event.metaKey) && event.key === "k") {
+      openQuickFind({
+        payload: {
```

Comment:
If you use `useReceiveMessage` on the other side you don't even need to send a payload. None of the things you're sending are used on the other side are they?